loan-agreement/src/main/java/de/weinbrecht/luc/bpm/architecture/loan/agreement/adapter/in/process/ApproveLoanAgreement.java

```diff
@@ -19,15 +18,8 @@ public class ApproveLoanAgreement {
 
     private final LoanAgreementStatusCommand loanAgreementStatusCommand;
 
-    @ZeebeWorker(type = LOAN_AGREEMENT_TASK, fetchVariables = LOAN_AGREEMENT_NUMBER)
-    public void handleJobFoo(final JobClient client, final ActivatedJob job) {
-        Long loanAgreementNumber = ((Number) job.getVariablesAsMap().get(LOAN_AGREEMENT_NUMBER)).longValue();
-        loanAgreementStatusCommand.accept(new LoanAgreementNumber(loanAgreementNumber));
-
-        client.newCompleteCommand(job.getKey())
-                .send()
-                .exceptionally( throwable -> {
-                    throw new CouldNotCompleteJobException(job, throwable);
-                });
+    @ZeebeWorker(type = LOAN_AGREEMENT_TASK, fetchVariables = LOAN_AGREEMENT_NUMBER, autoComplete = true)
```

Comment:
This might be nitpicking - but you don't need to specify "fetchVariables" here, this is done automatically by @ZeebeVariable (we tried to apply DRY as much as possible)

Reply:
Despite that - I really love the adjusted code - it is so much more readable :-) 👍

Reply:
Yes that's write.
I used both trying to keep the "visual sync" between the constants string variable (needed to write the variables) and the parameter. But yes, that's a bit overkill, I will adopt it. 

That you for your feedback! 🙂